src/peft/tuners/__init__.py

```diff
@@ -18,7 +18,7 @@
 # limitations under the License.
 
 from .adaption_prompt import AdaptionPromptConfig, AdaptionPromptModel
-from .lora import LoraConfig, LoraModel, LoftQConfig, LoraRuntimeConfig
+from .lora import LoraConfig, LoraModel, LoftQConfig, LoraPlusConfig, LoraRuntimeConfig
```

Comment:
Hmm, do we really need it here? I think it's enough to have this in the global PEFT namespace.

Reply:
Will try -- that seems like it applies to the other ones in there too though. What is different?

Reply:
True, maybe this was added in a refactor for backwards compatibility, I don't quite remember. Anyway, since the others are "tuners" but LoRA+ is for optimizers, I still think it's better not to add `LoraPlusConfig` here.

Reply:
Good point!